pages/index.js

```diff
@@ -1,11 +1,15 @@
 import MainColor from "@/components/MainColor";
+import ButtonNextPage from "@/components/ButtonNextPage";
+import Link from "next/link";
+import { StyledContainer } from "@/components/ButtonNextPage/styles";
 
-export default function HomePage() {
+export default function HomePage({ color, handleColorChange }) {
   return (
-    <>
-      <MainColor text="color1" id="color1" />
-      <MainColor text="color2" id="color2" />
-      <MainColor text="color3" id="color3" />
-    </>
+    <StyledContainer>
+      <MainColor color={color} handleColorChange={handleColorChange} />
+      <Link href="/suggestions-page">
+        <ButtonNextPage text="COLOR ME !" />
```

Comment:
Here's the little culprit. 😬 
Please note my comments in the corresponding [file](components/ButtonNextPage/index.js).